.buildkite/pipeline.yml

```diff
@@ -31,7 +31,11 @@ steps:
 
   - trigger: "kolibri-raspbian-image"
     label: ":raspberry-pi:"
-    build: *triggered_build_attributes
+    build: &triggered_build_attributes
```

Comment:
sorry, just curious - what's the difference between `*` and `&`?

Reply:
It's kind of (barely) like pointer syntax.

Before, the MacOS object was defining a [YAML anchor](https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html) with build properties to be used by all the trigger steps - `triggered_build_attributes`

But I commented it out, so the definition vanished. Now the definition lives in the `kolibri-raspbian-image` object